test/test-assert.h

```diff
@@ -22,8 +22,16 @@ class assertion_failure : public std::logic_error {
 
 void assertion_failure::avoid_weak_vtable() {}
 
+// We use a separate function (rather than throw directly from FMT_ASSERT) to
+// avoid GCC's -Wterminate warning when FMT_ASSERT is used in a destructor.
+inline void throw_assertion_failure (const char *message)
+{
+  throw assertion_failure(message);
+}
+
+
```

Comment:
Please remove extra empty line.

Reply:
Ooops. Thanks for your patience. Fixed with clang-format (which made the exact same edits you suggested).